docs/getting-started/connecting-to-wordpress.md

```diff
@@ -0,0 +1,110 @@
+
+{% code title="frontity.settings.js" %}
+```typescript
+const settings = {
+  ...,
+  packages: [
+    ...,
+    {
+      name: "@frontity/wp-source",
+      state: {
+        source: {
+          // Change this url to point to your WordPress API.
+          api: "https://test.frontity.io/wp-json/"
+        }
+      }
+    }
+  ]
+}
+```
+{% endcode %}
+
+Your site at http://localhost:3000 won't auto-update on this occasion as auto-updates only occur when you make a change to a file in the `packages` folder, so you will need to manually refresh the page in your browser.
+
+You should now see your own posts in the browser.
+
+## Updating the menu
+
+However the menu still has links to the demo content which will no longer work. You can change these to point to archives or pages on your own WordPress site. To do so locate the `@frontity/mars-theme` settings object and change the `menu` array to reflect the structure of your own site. So for example:
+
+{% code title="frontity.settings.js" %}
+```typescript
+const settings = {
+  ...,
+  packages: [
+    {
+      "name": "@frontity/mars-theme",
+      "state": {
+        "theme": {
+          "menu": [
+            [
+              // The link to your home page
+              "Home",
+              "/"
+            ],
+            [
+              // A link to a page
+              "About",
+              "/about/"
+            ],
+            [
+              // A link to the 'books' category
+              "Books",
+              "/category/books/"
+            ],
+            ...
+          ],
+          "featured": {
+            "showOnList": false,
+            "showOnPost": false
+          }
+        }
+      }
+    },
+    ...
+  ]
+}
+```
+{% endcode %}
+
+Again you should refresh the browser manually in order to see these changes.
+
+## Edit the starter theme
+
+You're now ready to start customising Frontity to render the content and appearance that you want for your site. This is where the fun begins.
+
+With Frontity you won't have your site code in the root folder as you would with `gatsbyjs` or `next.js`. Instead you develop Frontity packages and in order to personalize your site you'll edit a theme package.
+
+If you're just starting out with Frontity then the theme you should have selected when you ran `npx frontity create` will be `@frontity/mars-theme`. You can find it in `packages/mars-theme/`. Go ahead and edit any of its files while running Frontity and the site will refresh automatically.
+
+Move onto the [**Learning Frontity**](../learning-frontity/README.md) section to dig in to the details of Frontity and how you can work with it.
+
+> `npx frontity create` will install the `mars-theme` in `packages/` so the theme can be edited by the developer. However, this theme doesn't need to reside in `packages/` if you don't want to edit it. It can be installed directly with `npm` inside `node_modules/` and it will work exactly the same.
+
+## Production mode
```

Comment:
I wouldn't explain here the `npx frontity build` as it's only necessary for the deployment
I would just add a link to the deployment section with a final phrase like 
> Visit out [Deplyment Guide](deployment.md) to know more about how to deploy your project once is ready 

Also the `Edit the starter theme` doesn't really have to do with `connecting Frontity to Your WordPress Site` so I would remove this sections

What I would add here is a quick section about the `connect` function that can be used to connect your react components to the state (so they can access to the Wordpress info). Also it can be a good excuse to link to other parts of documentation such as: `state`, `router`, `connect API` and so on

Reply:
Are you talking about this: https://docs.frontity.org/api-reference-1/frontity#connect ?
If so I think that talking about the `connect` function might be going a bit deep a bit too soon for a getting started guide.